packages/create-novu-app/create-app.ts

```diff
@@ -0,0 +1,231 @@
+/* eslint-disable */
```

Comment:
In general I needed to add this in a lot of places to allow the commit to pass, they should be removed as devs work on given files